servers_v6.json

```diff
@@ -87,10 +87,6 @@
     "name": "Nydus",
     "address": ["v6.mindustry.nydus.app:6566"]
   },
-  {
-    "name": "MD Community",
-    "address": ["62.109.31.34", "185.13.47.140:1001"]
-  },
```

Comment:
leave mindurka, .ru just technically changed name to remove their own reputation ,